lighthouserc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "ci": {
+    "collect": {
+      "url": ["http://localhost:4000/"],
```

Comment:
I'd be keen on getting other URLs here in a follow up PR.

Reply:
Agreed! I (apparently mistakenly) thought Lighthouse CI crawled linked page, but the [`url` docs](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/configuration.md#url) don’t mention that.

Reply:
I think visits all your pages if you're just using [`staticDistDir`](https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/configuration.md#staticdistdir) and not `url`? It's not super clear to me from the documenation.

>  --autodiscoverUrlBlocklist A URL to not include when autodiscovering urls from staticDistDir.
                             Use this flag multiple times to filter multiple URLs.